app/scripts/lib/snap-keyring/utils/isBlockedUrl.test.ts

```diff
@@ -10,6 +10,7 @@ describe('isBlockedUrl', () => {
     allowedEvents: [],
   });
   const phishingController = new PhishingController({
+    // @ts-expect-error TODO: Resolve/patch mismatch between messenger types
```

Comment:
Note: This is caused by the major version difference in the `base-controller` versions used in `phishing-controller` (v6) vs. extension (v5). 

The only reason we're not seeing the same error in [`metamask-controller.js`](https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/metamask-controller.js#L839) is because the file hasn't been converted to TypeScript.